.rspec

```diff
@@ -1,2 +1,2 @@
---format documentation
 --color
+--format documentation
```

Comment:
why did you change this?

Reply:
Shoot - I didn't mean to commit that. I converted the module to be PDK compatible for my own local testing during development and I accidentally added this file. PDK changed a ton of stuff and I'm not familiar enough with Travis or the testing configuration in use on this module so I didn't add the other changes. I'll revert this for now.